sql/identities/identities.go

```diff
@@ -26,11 +49,25 @@ func SetMalicious(db sql.Executor, nodeID types.NodeID, proof []byte, received t
 	if err != nil {
 		return fmt.Errorf("set malicious %v: %w", nodeID, err)
 	}
+	ids, err := EquivocationSet(db, nodeID)
+	if err != nil {
+		return fmt.Errorf("get equivocation set for %v: %w", nodeID, err)
+	}
+	for _, id := range ids {
+		sql.AddToSet(db, "malicious", id[:])
+	}
```

Comment:
Based on your discussion with @dshulyak, shouldn't this be:
```suggestion
	sql.AddToSet(db, "malicious", id[:])
```

provided `SetMalicious` is called for every `ID` in the equivocation set? Note: calling `SetMalicious` is not yet implemented for ATX V2 AFAIR: https://github.com/spacemeshos/go-spacemesh/blob/83b04992a06d8204c8588f52c97fe187680f4499/activation/malfeasance2.go#L13-L16